app/scripts/controllers/add-events.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (Api, $scope, $routeParams, $location) {
   Api.Patients.get({ id: $routeParams.patientId }, success, error)
 
   $scope.events = [
-    { code: 'sample-event', display: 'Sample Event', formName: 'FormBuilderaddEventSampleEvent' }
+    { code: 'immunisation-notification', display: 'Immunisation Notification', formName: 'FormBuilderAddImmunisationNotification' }
```

Comment:
FormBuilderAddImmunisationNotification is supposed to be initialise as a const above this file to point to your app/scripts/directives/add-events/add-event/forms/immunisation-notification.json file